xikolo-ios/VideoViewController.swift

```diff
@@ -15,24 +15,55 @@ class VideoViewController: UIViewController {
     @IBOutlet weak var containerVideoView: UIView!
     @IBOutlet weak var titleLabel: UILabel!
     @IBOutlet weak var descriptionTextView: UITextView!
+    @IBOutlet weak var openSlidesButton: UIButton!
+    @IBOutlet weak var previousItemButton: UIButton!
+    @IBOutlet weak var nextItemButton: UIButton!
+    @IBOutlet weak var containerHeighConstraint: NSLayoutConstraint!
     
-    var courseItem: CourseItem! {
+    var courseItem: CourseItem!
+    var video: Video? = nil {
         didSet {
-            updateUI()
+            videoDidLoad()
+            performSegueWithIdentifier("EmbedAVPlayer", sender: video)
         }
     }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        containerHeighConstraint.constant = self.view.frame.size.width * (9/16)
+        let videoIncomplete = courseItem.content as! Video
+        VideoHelper.syncVideo(videoIncomplete).onSuccess { videoComplete in
+            self.video = videoComplete
+        }
+    }
+
+    func videoDidLoad() {
+        titleLabel.text = courseItem?.title
```

Comment:
You have defined `courseItem` as `CourseItem!`, an implicitly unwrapped optional. If there's a need to check for `nil` here (using `?`), it should be a real optional. If there isn't, remove the `?`.